### DISCUSSION TITLE
Welcome to Waifu Motivator Plugin Discussions!

### DISCUSSION THREAD

**Discussion by zaerald** [waifu-motivator/waifu-motivator-plugin#283] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Unthrottled** [comment]
I'll break the ice.

For starters, I spend way more time decorating all of my tech tools with anime, than I do actually watching anime. _I think I might have a problem_. I've got some big and exciting things that I am working on and am looking forward to being able to share it with the community! 

 I also think I might have garbage taste, because I just finished Konosuba and I think Aqua is best girl.

![Aqua](https://media1.tenor.com/images/f97bf8e226d5e46dce0220a375b6e59b/tenor.gif?itemid=14708521)

:joy:
